### DISCUSSION TITLE
Lift Z

### DISCUSSION THREAD

**Discussion by Durahl** [supermerill/SuperSlicer#2576] [Q&A] [answered]
Greetings!

I'm currently trying to 3D Print a Tri-Tone Plaque but am having issues with the nozzle getting contaminated by the old color while travelling from one section to the next.

Example:
![IMAGE](https://i.imgur.com/lIuCTQQ.png)
In the above picture the Black Filament gets extruded first > A minor black purge happens > A major white Filament purge happens > The white Filament starts extruding and while moving from one letter to the next the Nozzle starts picking up black Filament.

I tried to have it Lift Z found in the Extruder > Retraction Settings but its not doing any Lifting when moving from one Letter to the Next:
![IMG](https://i.imgur.com/HHIgmai.png)
Am I looking at the wrong setting? 🤔

**Comment by supermerill** [comment, marked as answer]
lift only happen when a retraction happens.
check you didn't select `retract only when crossing perimeter` in infill tab

btw, you don't ned the "enforce on first layer" if you set the "above z" to 0

**Comment by Durahl** [comment]
@supermerill Yea... The `Infill > Only retract when crossing perimeters` was causing me grief!
Thanks a lot =)
